plugins/hydra_nevergrad_sweeper/hydra_plugins/hydra_nevergrad_sweeper/core.py

```diff
@@ -146,16 +149,18 @@ def __init__(
         num_workers: int,
         noisy: bool,
         maximize: bool,
+        seed: int,
         version: int,
     ):
-        self.config: tp.Optional[DictConfig] = None
-        self.launcher: tp.Optional[Launcher] = None
+        self.config: Optional[DictConfig] = None
+        self.launcher: Optional[Launcher] = None
         assert version == 1, "Only version 1 of API is currently available"
         self.job_results = None
         self.optimizer = optimizer
         self.noisy = noisy
         self.budget = budget
         self.num_workers = num_workers
+        self.seed = seed
```

Comment:
and you are not handling -1 :)

Reply:
it was handled, in the `sweep` method, but now I can remove an extra line `if self.seed != -1`^^